packages/components/src/combobox-control/test/index.js

```diff
@@ -0,0 +1,141 @@
+	{ name: 'Universal Coordinated Time', abbreviation: 'UTC' },
+	{ name: 'European Central Time', abbreviation: 'ECT' },
+	{ name: '(Arabic) Egypt Standard Time', abbreviation: 'ART' },
+	{ name: 'Eastern African Time', abbreviation: 'EAT' },
+	{ name: 'Middle East Time', abbreviation: 'MET' },
+	{ name: 'Near East Time', abbreviation: 'NET' },
+	{ name: 'Pakistan Lahore Time', abbreviation: 'PLT' },
+	{ name: 'India Standard Time', abbreviation: 'IST' },
+	{ name: 'Bangladesh Standard Time', abbreviation: 'BST' },
+	{ name: 'Vietnam Standard Time', abbreviation: 'VST' },
+	{ name: 'China Taiwan Time', abbreviation: 'CTT' },
+	{ name: 'Japan Standard Time', abbreviation: 'JST' },
+	{ name: 'Australia Central Time', abbreviation: 'ACT' },
+	{ name: 'Australia Eastern Time', abbreviation: 'AET' },
+	{ name: 'Solomon Standard Time', abbreviation: 'SST' },
+	{ name: 'New Zealand Standard Time', abbreviation: 'NST' },
+	{ name: 'Midway Islands Time', abbreviation: 'MIT' },
+	{ name: 'Hawaii Standard Time', abbreviation: 'HST' },
+	{ name: 'Alaska Standard Time', abbreviation: 'AST' },
+	{ name: 'Pacific Standard Time', abbreviation: 'PST' },
+	{ name: 'Phoenix Standard Time', abbreviation: 'PNT' },
+	{ name: 'Mountain Standard Time', abbreviation: 'MST' },
+	{ name: 'Central Standard Time', abbreviation: 'CST' },
+	{ name: 'Eastern Standard Time', abbreviation: 'EST' },
+	{ name: 'Indiana Eastern Standard Time', abbreviation: 'IET' },
+	{ name: 'Puerto Rico and US Virgin Islands Time', abbreviation: 'PRT' },
+	{ name: 'Canada Newfoundland Time', abbreviation: 'CNT' },
+	{ name: 'Argentina Standard Time', abbreviation: 'AGT' },
+	{ name: 'Brazil Eastern Time', abbreviation: 'BET' },
+	{ name: 'Central African Time', abbreviation: 'CAT' },
+];
+const mapTimezoneOption = ( tz ) => ( {
+	value: tz.abbreviation,
+	label: tz.name,
+} );
+
+const timezoneOptions = timezones.map( mapTimezoneOption );
+const defaultLabelText = 'Select a timezone';
+const getLabel = () => screen.getByText( defaultLabelText );
+const getInput = () => screen.getByRole( 'combobox' );
+const getRenderedOptions = () => screen.getAllByRole( 'option' );
+
+describe( 'ComboboxControl', () => {
+	// Use real timers to avoid timeout errors from userEvent
+	beforeEach( () => {
+		jest.useRealTimers();
+	} );
+
+	const TestComboboxControl = ( props ) => {
+		const [ value, setValue ] = useState( null );
+		return (
+			<>
+				<ComboboxControl
+					{ ...props }
+					value={ value }
+					label={ defaultLabelText }
+					options={ timezoneOptions }
+					onChange={ setValue }
+				/>
+				<p data-testid="value-output">{ value }</p>
+			</>
+		);
+	};
+
+	it( 'should render', () => {
+		render( <TestComboboxControl /> );
+
+		const input = getInput();
+		expect( input ).toBeVisible();
+	} );
+
+	it( 'should render with visible label', () => {
+		render( <TestComboboxControl /> );
+		const label = getLabel();
+		expect( label ).toBeVisible();
```

Comment:
Seeing this test, it makes me think that by default the `<TestComboboxControl />` should NOT add a `label` to `ComboboxControl` — it should be possible to pass `label` as a prop to `TestComboboxControl`

Reply:
Cool. Along with this change, I also updated `getLabel()` to accept any label text, since `defaultLabelText` won't be the automatic default any longer. This way we can pass any label we need for a test, and then make sure it's rendering that text correctly.